source/common/runtime/runtime_features.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/runtime/runtime_features.h"
+
+namespace Envoy {
+namespace Runtime {
+
+// Add additional features here to enable the new code paths by default.
+//
+// These features should not be overridden with run-time guards without a bug
+// being filed on github as once high risk features are true by default, the
+// old code path will be removed with the next release.
+constexpr const char* runtime_features[] = {
+    // Enabled
+    "envoy.reloadable_features.test_feature_true",
+    // Disabled
+    // "envoy.reloadable_features.test_feature_false",
```

Comment:
Do we mean for this to be commented out here? If this is a documented way of defaulting to disabled, can you add more comments?

Reply:
I was thinking at some point it'd be nice to have all flags documented in that file, so if folks want to preemptively flip they can do so.

I'll remove it and wait until we have utils and see if it still makes sense